### DISCUSSION TITLE
Organizations default to `View as: Public` rather than Member

### DISCUSSION THREAD

**Discussion by donatj** [community/community#38684] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am a member of a handful of organizations. 

Recently some of my organizations started showing as `View as: Public` every time I visit them.

![image](https://user-images.githubusercontent.com/133747/200883528-f3beb3ac-ac43-4def-ba20-5ccbbc01fe48.png)

This means I have to change to `View as: Member` _every single time_ I want to see my private repos. It's just kind of a pain.  

I *am* a member, for most of them I am the owner; I shouldn't have to pick `View as: Member` every time.

This seems to have started a couple weeks ago. I've scoured the Organization config for a setting and can't find anything which leads me to believe this is a bug.

**Comment by softfrog** [comment]
Same here - this is a pain. I need to switch between repos often and now I have to view as member every time before I can see the repo list.

**Reply by HamzatAbdulazeez** [reply]
Check your code and might be the processing

**Reply by donatj** [reply]
@HamzatAbdulazeez What?

**Comment by mwaibel-go** [comment]
Maybe as a general rule: Don’t change the UI without ~~any way~~ an obvious hint to get back old behaviour.

For the others: While it’s not obvious, you can use the search bar in the public view to search private repos.

**Comment by pmenke-de** [comment]
After some browsing through the github changelogs, i've found this: https://github.blog/changelog/2022-04-20-organization-profile-updates-member-only-readmes-and-pinned-private-repositories/

In the referenced [docs page](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile) i've found that you need to

1. create a repository `.github-private` in your organization
2. create a file `/profile/README.md` in that repo

The "View as" property will then default to "member" for members 😵‍💫

**Comment by softfrog** [comment]
Thanks for that lead - I solved it just by pinning the repos, no need for a
.github-private

**Reply by donatj** [reply]
Sure enough, pinning a single private repo seems to fix the issue. This is so dumb - who came up with this?

**Comment by alexandervauer** [comment]
Maybe they could add a setting for this under Settings>Access>Organizations somewhere ¯\_(ツ)_/¯

**Reply by mwaibel-go** [reply]
I’m not even sure why my organization even gets to choose this. I think it should be up to me to configure my view of the organization.

**Reply by alexandervauer** [reply]
Yeah, I agree on that one

**Reply by donatj** [reply]
I mean for what it's worth I don't see why you would ever not want the members just to see it as members with the organization repos visible.

**Reply by michael-andreuzza** [reply]
2024 and the issue still going on....this makes no sense. I can't even pin repos...
